enclave-runtime/src/sidechain_impl.rs

```diff
@@ -145,22 +144,28 @@ where
 {
 	log::info!("[Aura] Executing aura for slot: {:?}", slot);
 
-	let mut aura =
-		Aura::<_, _, SB, PEnvironment, _>::new(authority, ocall_api.clone(), proposer_environment)
-			.with_claim_strategy(SlotClaimStrategy::Always)
-			.with_allow_delayed_proposal(true);
+	let mut aura = Aura::<_, _, SB, PEnvironment, _>::new(
+		authority.clone(),
+		ocall_api.clone(),
+		proposer_environment,
+	)
+	.with_claim_strategy(SlotClaimStrategy::Always)
+	.with_allow_delayed_proposal(true);
 
 	let (blocks, xts): (Vec<_>, Vec<_>) =
 		PerShardSlotWorkerScheduler::on_slot(&mut aura, slot, shards)
 			.into_iter()
 			.map(|r| (r.block, r.parentchain_effects))
 			.unzip();
 
-	prepare_and_send_xts_and_block::<_, SB, _, _>(
+	ocall_api.send_sidechain_blocks(blocks)?;
```

Comment:
Sending the sidechain blocks is now done by just calling the corresponding method on `ocall_api`